version-map.txt

```diff
@@ -1,4 +1,2 @@
 # List of Sopel versions to build:
-v7.0.0
-v7.0.4
-v7.0.4
+v7.0.6
```

Comment:
Since v7.0.5 never got built, let's have Travis build+push v7.0.5 as well, for posterity's sake. I plan to update the comments in this file to ensure that the proper versions get added/removed for future version bump PRs.
```suggestion
v7.0.5
v7.0.6
```
the instructions will probably be something like:
for a version bump where the current version is > 1 step away from the last built image, either
a) submit multiple PRs, one for each step; this way travis will build+deploy each merged PR (i.e., version)
b) change this file to include each version to be built